File: api/src/main/java/com/ruby/rubia_server/core/enums/WhatsAppInstanceStatus.java
package com.ruby.rubia_server.core.enums;

public enum WhatsAppInstanceStatus {
    NOT_CONFIGURED("Não Configurado"),
    CONFIGURING("Configurando"),
    AWAITING_QR_SCAN("Aguardando Escaneamento QR"),
    CONNECTING("Conectando"),
    CONNECTED("Conectado"),
    DISCONNECTED("Desconectado"),
    ERROR("Erro"),
    SUSPENDED("Suspenso");

    private final String displayName;

    WhatsAppInstanceStatus(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: api-dotnet/Rubia.Server/Enums/WhatsAppInstanceStatus.cs
using System.ComponentModel;

namespace Rubia.Server.Enums;

public enum WhatsAppInstanceStatus
{
    [Description("Não Configurado")]
    NotConfigured,
    
    [Description("Configurando")]
    Configuring,
    
    [Description("Aguardando QR Code")]
    AwaitingQrScan,
    
    [Description("Conectando")]
    Connecting,
    
    [Description("Conectado")]
    Connected,
    
    [Description("Desconectado")]
    Disconnected,
    
    [Description("Erro")]
    Error,
    
    [Description("Suspenso")]
    Suspended
}